src/App.tsx

```diff
@@ -1,204 +1,69 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { ErrorNotification } from './components/ErrorNotifications';
+import { Errors } from './types/Errors';
+import { FilterStatus } from './types/FilterStatus';
+import { getFilteredTodos } from './utils/getFilteredTodos';
 
 export const App: React.FC = () => {
+  const [todos, setTodods] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(Errors.DEFAULT);
+  const [filterStatus, setFilterStatus] = useState(FilterStatus.ALL);
+
+  const handleResetErrorMessage = () => {
+    setErrorMessage(Errors.DEFAULT);
+  };
+
+  useEffect(() => {
+    setErrorMessage(Errors.DEFAULT);
+    setIsLoading(true);
+    getTodos()
+      .then(setTodods)
+      .catch(() => setErrorMessage(Errors.LOADING_TODOS))
+      .finally(() => setIsLoading(false));
+    setTimeout(() => {
+      setErrorMessage(Errors.DEFAULT);
+    }, 3000);
```

Comment:
In this case you call `setTimeout()` on every instance, even when you don't have an error.